components/connection/ConnectionList.tsx

```diff
@@ -16,7 +16,8 @@ import { fetcher } from '@lib/ui/utils';
 import Loading from '@components/Loading';
 import { errorToast } from '@components/Toaster';
 import type { ApiError, ApiSuccess } from 'types';
-import { Badge } from 'react-daisyui';
+import { isConnectionActive } from '@lib/utils';
```

Comment:
Unused import can be removed from here, also saw it in couple of other places.

Reply:
Removed from all places.